packer/ansible/roles/remoterepos/tasks/main.yml

```diff
@@ -1,46 +1,152 @@
 ---
+# This role supports developers who want to pull code from one or more repositories forked into their github organization.
```

Comment:
can you please write a **usage** guide on the top, to tell which parameters does this new playbook accept ?
- organization 
- branch
- repos ?